internal/entities/tree.go

```diff
@@ -19,3 +19,12 @@ type Tree struct {
 	Longitude      float64
 	WateringStatus WateringStatus
 }
+type TreeCreate struct {
+	TreeClusterID *int32
+	Readonly      bool
+	PlantingYear  int32
+	Species       string
+	Number        string
+	Latitude      float64
+	Longitude     float64
+}
```

Comment:
Do we really need this?

Reply:
For `TreeCluster`, we have both `TreeClusterCreate` and `TreeClusterUpdate` structs for creating and updating, so I followed the same approach for Tree. What do you mean—are you suggesting we use the `TreeCreateRequest` directly in the handler without mapping it to another struct that has the same values?

Reply:
Okay, that's a point... I had a different idea but the more I think about it the smarter it is to do it this way :+1: